src/components/club/listing/ClubTitle.tsx

```diff
@@ -0,0 +1,61 @@
+import Chip from '@mui/material/Chip';
+import Image from 'next/image';
+import { type RouterOutputs } from '@src/trpc/shared';
+import JoinButton from '../JoinButton';
+
+const ClubTitle = async ({
+  club,
+}: {
+  club: NonNullable<RouterOutputs['club']['getDirectoryInfo']>;
+}) => {
+  return (
+    <section
+      id="club-tile"
+      className="w-full rounded-lg flex flex-col items-start md:flex-row justify-between gap-4 mb-0 mt-2"
+    >
+      {club.profileImage && (
+        <Image
+          src={club.profileImage}
+          alt={club.name + ' logo'}
+          width={128}
+          height={128}
+          className="rounded-lg"
+        />
+      )}
+      <div className="flex flex-col flex-grow">
+        {club.name && (
+          <h1
+            className={`font-display font-bold text-slate-800 ${
```

Comment:
I prefer the semibold font weight; I think bold is too heavy.

```suggestion
            className={`font-display font-semibold text-slate-800 ${
```
If you do change this, you'll need to add the '600' weight to line 19 of `/app/layout.tsx`